nextflow.config

```diff
@@ -1,85 +1,145 @@
-// =================================================================
-// nextflow.config is the main configuration script for a nextflow pipeline
-// Should contain the following sections:
-	// A manifest for defining workflow metadata
-	// Mandated minimal version of Nextflow required to run pipeline
-	// Default workflow parameter definitions
-	// Shell behaviour settings for the workflow
-	// Execution reports
-	// Configuration profiles
-	// Default resource definitions for processes
-
-// Examples are included for each section. Remove them and replace
-// with project-specific code. For more information see:
-// https://www.nextflow.io/docs/latest/index.html.
-//
-// ===================================================================
-
-// Define some metadata for publishing pipeline on GitHub
-// See https://www.nextflow.io/docs/latest/config.html#scope-manifest
+// Workflow details
 manifest {
-	author = 'Georgie Samaha'
-	name = 'Nextflow_DSL2_template-nf'
-	description = 'Template for creating Nextflow workflows'
-	homePage = 'https://github.com/Sydney-Informatics-Hub/Nextflow_DSL2_template'
-}
+    
+  author = ' Tracy Chew, Cali Willet, Nandan Deshpande, Sydney Informatics Hub, University of Sydney'
+  name = 'SomaticShortV-nf'
+  description = 'GATK somatic short variant calling pipeline'
+  homePage = 'https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf'
+    }
 
 // Mandate a minimum version of nextflow required to run the pipeline
-// See https://www.nextflow.io/docs/latest/config.html#nextflow-version
-nextflowVersion = '!>=20.07.1'
+nextflowVersion     = '!>=20.07.1'
 
-// Set default workflow parameters
-// See https://www.nextflow.io/docs/latest/config.html#scope-params 
+// Set default parameters
 params {
-	help	= false
-	outDir	= 'results'
-	input	= false
+  version                       = '1.0'
+  help                          = false
+  input                         = null
+  ref                           = false
+  intervals                     = false
+  outDir		        = './results'
+
+
+  // Intervals for 'Scatter-Gather'
+  intervalList          = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n']
```

Comment:
As per my comment above on scatter files. Why have you used abc instead of names that correspond to the interval? Consider how this might be useful to someone else running your workflow. If the a,b,c convention comes from somewhere and has meaning, could you explain this in a comment and add a reference.

Reply:
Same response as above 

I had made the following scatter lists

B. GATK-style .list or .intervals from [Intervals and interval lists](https://gatk.broadinstitute.org/hc/en-us/articles/360035531852-Intervals-and-interval-lists)
I have now switched to the
A. Picard-style .interval_list at [Intervals and interval lists ](https://gatk.broadinstitute.org/hc/en-us/articles/360035531852-Intervals-and-interval-lists)which are available at /g/data/er01/SIH-HPC-WGS/Reference/BQSR_apply_intervals_13
The interval lists from modules are now removed